scripts/rmc.py

```diff
@@ -255,6 +258,83 @@ def run_goto_instrument(input_filename, output_filename, args, verbose=False, dr
 # Generates a C program from a goto program
 def goto_to_c(goto_filename, c_filename, verbose=False, dry_run=False):
     return run_goto_instrument(goto_filename, c_filename, ["--dump-c"], verbose, dry_run=dry_run)
+    
+# Fix remaining issues with output of --gen-c-runnable
+def gen_c_postprocess(c_filename, dry_run=False):
+    if not dry_run:
+        with open(c_filename, "r") as f:
+            lines = f.read().splitlines()
+
+        # Remove header line
+        lines.pop(0)
```

Comment:
If you run it like this
```
goto-instrument --dump-c be.goto be.c
```

instead of like that 
```
goto-instrument --dump-c be.goto > be.c
```

You don't have the line you need to pop

Reply:
Done.